301d58/code-challenges/includes/challenges-13.test.js

```diff
@@ -0,0 +1,309 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named firstLetters that takes in an array of strings and returns an array containing only the first letter of each string.
+
+For example, ['this is great :)', 'wow', 'whyyyyyy :(', ':)))))'] returns ['t', 'w', 'w', ':']
+------------------------------------------------------------------------------------------------ */
+
+const firstLetters = (arr) => {
+  let newArr = [];
+  arr.forEach(string => {
+    newArr.push(string[0]);
+  });
+  return newArr;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named findHappiness that takes in an array of strings and returns an array containing only the strings from the input array that contain ":)".
+
+For example, ['this is great :)', 'wow', 'whyyyyyy :(', ':)))))'] returns ['this is great :)', ':)))))']
+------------------------------------------------------------------------------------------------ */
+
+const findHappiness = (arr) => {
+  let newArr = [];
+  arr.forEach(string => {
+    if(string.includes(':)')) {
+      return newArr.push(string);
+    }
+  });
+  return newArr;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 3
+
+Write a function named standardizePhoneNumbers that takes in an array of phone number strings in (XXX) XXX-XXXX
+format and returns an array with the phone number strings in XXXXXXXXXX format.
+
+For example, (123) 456-7890 returns 1234567890
+------------------------------------------------------------------------------------------------ */
+
+const standardizePhoneNumbers = (arr) => {
+    return arr.map(value => {
+    return value.replace(/[\s()-]/g, '');
```

Comment:
This definitely works, you could have tried to select everything that isn't a number